apps/arweave/src/ar_coordination.erl

```diff
@@ -379,7 +399,7 @@ refresh([], State) ->
 	State;
 refresh([Peer | Peers], State) ->
 	NewState = case ar_http_iface_client:get_cm_partition_table(Peer) of
-		{ok, PartitionList} ->
+		{ok, PartitionList} when length(PartitionList) > 0 ->
```

Comment:
In this case user will see there is no connection to that node
Original code will make sure that user sees that there is connection to node, but no partitions there

Reply:
Got it. Reason I added that clause was with the new code the node only returns partitions if it is configured to mine - so it may have storage_modules defined, but if mining is disabled, or if the node recently restarted and is still initializing, it will see no partitions.

I guess that' sfine though... if mining is disabled then that node shouldn't receive H1s anyways since it won't be able to compute the H2. And if it's in the middle of initializing, should be fine too since it will be polled again in 60s at which point it will have partitions.

Ok, I'll remove the `when` clause!